websites/R/RhysBOT/presence.ts

```diff
@@ -0,0 +1,38 @@
+const presence = new Presence({
+		clientId: "658765364439810048"
+	}),
+	date = Date.now();
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "Logo",
+			details: "Exploring",
+			startTimestamp: date,
+			buttons: [
+				{
+					label: "View Page",
+					url: document.location.href
```

Comment:
you should add buttons when relevant and with appropriate labels, not on the initial declaration